src/components/provider/UALProvider.js

```diff
@@ -282,6 +286,34 @@ export class UALProvider extends Component {
     return loggedIn.length ? loggedIn[0] : false
   }
 
+  /**
+   * Checks if the saved browser session has passed the UALInvalidateAt date and clear the cache if true
+   * @method
+   * @param {string} type - UALLoggedInAuthType value
+   * @param {string} invalidate - UALInvalidateAt value in string formatted date
+   * @return {string|undefined} - UALLoggedInAuthType value or undefined if no longer valid
+   */
+  checkForInvalidatedSession = (type, invalidate) => {
+    if (type && invalidate && new Date(invalidate) <= new Date()) {
+      this.clearCache();
```

Comment:
Should we call ``this.logout()`` as well?

Reply:
Since the login hasn't been started and `activeAuthenticator` isn't set yet at this point, logout causes an error.